liquidity-module/index.md

```diff
@@ -321,17 +346,50 @@ For this example command, be sure to:
 
 To confirm the pool has been created:
 
-- Visit [https://api.testnet.cosmos.network/tendermint/liquidity/v1beta1/pools](https://api.testnet.cosmos.network/tendermint/liquidity/v1beta1/pools)
+- Visit [https://api.testnet.cosmos.network/cosmos/liquidity/v1beta1/pools](https://api.testnet.cosmos.network/cosmos/liquidity/v1beta1/pools)
+
+**Tip:** Verify the pool created on terminal:
+```bash
+gaiad query liquidity pools
+```
+
+After you successfully query the pool you should see an output similar to:
+```markdown
+- id: "1"
+  pool_coin_denom: poolF4E2371BB7E34567B5A91A44808B8470BFAE0ABD606D707A8F1D59A3EB164816
+  reserve_account_address: cosmos17n3rwxahudzk0ddfrfzgpzuywzl6uz4alp5z2l
+  reserve_coin_denoms:
+  - ibc/32023ECF96BB757261CB59A37F9013D012969795D12D24FCCE50CBE5F879C920
+  - uphoton
+  type_id: 1
+  .
+  .
+  .
+- id: "6"
+  pool_coin_denom: poolFE384B6C9AE769A67754EAD275832A5A6DAA6E2C769A1E1D281E596DD47AA36D
+  reserve_account_address: cosmos1lcuykmy6ua56va65atf8tqe2tfk65m3v58zq6n
+  reserve_coin_denoms:
+  - ibc/2A399B0A1E83C2929B7C07E12B86F72EF4B4252ECB95A895BB51C2C00A106370
+  - uphoton
+  type_id: 1
+```
 
+You can also query a specific pool by `pool id` using the following command:
+```bash
+gaiad query liquidity pool 6
+```
+Make sure to replace `6` with your `pool id`
```

Comment:
```suggestion

Make sure to replace `6` with your pool id.
```

Reply:
in this case, we don't want to use tick marks around `pool id` because it's not a literal code. let's always leave a blank line before and after code fences